README.rst

```diff
@@ -1,116 +1,23 @@
 Meteoroid CLI
 =============
 
+.. image:: https://img.shields.io/pypi/v/meteoroid-cli
+     :target: http://badge.fury.io/py/meteoroid-cli
+.. image:: https://img.shields.io/github/license/OkinawaOpenLaboratory/fiware-meteoroid-cli?color=blue
+     :target: https://github.com/OkinawaOpenLaboratory/fiware-meteoroid-cli/blob/master/LICENSE
```

Comment:
Change the target from the full path to *./LICENSE*

Reply:
I updated.